js/classes.js

```diff
@@ -114,3 +114,9 @@ window.onload = () => {
   document.getElementById('contact').style.display = 'none';
   this.generateBooks();
 };
+
+const date = document.getElementById('date');
+// eslint-disable-next-line no-undef
+const DateTime = luxon.DateTime.now();
+// eslint-disable-next-line no-undef
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.